### ISSUE TITLE
Creating the Storybook framework

### ISSUE THREAD

**Issue by tobiu** [neomjs/neo#5132]
This should most likely be the starting point of this project. We can most likely break it down into smaller tickets.

We need to follow the official guide:
https://storybook.js.org/docs/contribute/framework

and analyse the already existing implementations:
https://github.com/storybookjs/storybook/tree/next/code/frameworks

e.g. the ember implementation might be work a closer look, since it seems to not be template-driven.

**Comment by tobiu** [comment]
More input from @valentinpalkovic:

We do not need to clone or fork the entire storybook repo.

Copying the code from one of the existing storybook frameworks into a new repo is sufficient. E.g.:
https://github.com/storybookjs/storybook/tree/next/code/frameworks/react-webpack5

Then we need to adjust the package name inside the package.json and can use this custom framework package inside storybook.

Once the setup is in place, the `renderToCanvas` method is key. E.g.:
https://github.com/storybookjs/storybook/blob/next/code/renderers/react/src/renderToCanvas.tsx

Out of scope for this ticket => https://github.com/neomjs/neo/issues/5133

**Comment by neo-opus-ada** [comment]
Closed not-planned with parent epic #5010 (operator-directed; Storybook does not fit Neo's app-worker + JSON-vdom architecture — a purpose-built Neo playground is the future direction).
